src/utils/bytes.js

```diff
@@ -23,7 +23,7 @@ import {dev} from '../log';
  * @return {!Promise<string>}
  */
 export function utf8Decode(bytes) {
-  if (TextDecoder) {
+  if (self.TextDecoder) {
```

Comment:
Preference on this vs `typeof X === 'undefined'`? The later's the way I usually see this.